crates/sui-deepbook-indexer/src/migrations/00000000000000_diesel_initial_setup/up.sql

```diff
@@ -178,4 +180,22 @@ CREATE TABLE IF NOT EXISTS sui_error_transactions
     failure_status              TEXT         NOT NULL,
     package                     TEXT         NOT NULL,
     cmd_idx                     BIGINT
-);
\ No newline at end of file
+);
+
+CREATE TABLE IF NOT EXISTS pools
+(
+    id                          SERIAL       PRIMARY KEY,
+    pool_id                     TEXT         NOT NULL,
+    pool_name                   TEXT         NOT NULL,
+    base_asset_id               TEXT         NOT NULL,
+    base_asset_decimals         BIGINT       NOT NULL,
+    base_asset_symbol           TEXT         NOT NULL,
+    base_asset_name             TEXT         NOT NULL,
+    quote_asset_id              TEXT         NOT NULL,
+    quote_asset_decimals        BIGINT       NOT NULL,
+    quote_asset_symbol          TEXT         NOT NULL,
+    quote_asset_name            TEXT         NOT NULL,
+    min_size                    BIGINT       NOT NULL,
+    lot_size                    BIGINT       NOT NULL,
+    tick_size                   BIGINT       NOT NULL,
```

Comment:
same question for smaller int types: https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-INT